Flow.Launcher.Core/Plugin/PluginsLoader.cs

```diff
@@ -21,16 +21,19 @@ public static List<PluginPair> Plugins(List<PluginMetadata> metadatas, PluginsSe
             var dotnetPlugins = DotNetPlugins(metadatas);
             
             var pythonEnv = new PythonEnvironment(metadatas, settings);
+            var pythonV2Env = new PythonV2Environment(metadatas, settings);
             var tsEnv = new TypeScriptEnvironment(metadatas, settings);
             var jsEnv = new JavaScriptEnvironment(metadatas, settings);
             var pythonPlugins = pythonEnv.Setup();
+            var pythonV2Plugins = pythonV2Env.Setup();
             var tsPlugins = tsEnv.Setup();
             var jsPlugins = jsEnv.Setup();
             
             var executablePlugins = ExecutablePlugins(metadatas);
```

Comment:
Should we not also instantiate the ExecutablePluginV2?

Reply:
Yeah true